staff/acct/sorry/sorry

```diff
@@ -1,156 +1,202 @@
+            return True
+
+
+def kerberos_destroy():
+    return subprocess.call(['kdestroy'])
+
+# Do all kinds of logging, like sending e-mails or writing to files
+# Change here if more crazy thingy need to be added
+
+
+def log(root_staffer, sorried_user, reason):
+    # Send log files only for now
+    send_log_mail(root_staffer, sorried_user, reason)
+
+# TODO: Do we want a log file or log email?
+
+
+def write_log_file(root_staffer, sorried_user):
+    #
+    with open(SORRY_LOG, 'a') as log_file:
+        # Date style matters...?
+        log_file.write('{0} - {1} {2}'.format(str(datetime.datetime.now()), root_staffer, sorried_user))
+
+
+def send_log_mail(root_staffer, sorried_user, reason):
+    # Use root@ for now
+    email_from = 'Open Computing Facility <root@ocf.berkeley.edu>'
+    email_subject = '[OCF] Account disabled'
+    email_body = textwrap.dedent('''
+        This is an automated email sent by OCF sorry script. This is the log of a recent sorry operation:
+        On "{3}", account "{0}" has been disabled by "{1}" for reason:
+        "{2}".
+        '''.format(sorried_user, root_staffer, reason, str(datetime.datetime.now())))
+    ocflib.misc.mail.send_mail('root@ocf.berkeley.edu', email_subject, email_body, cc=None, sender=email_from)
+
+
+def send_sorry_mail(sorried_user_email, username, reason):
+    email_subject = '[OCF] Account disabled'
+    email_body = '''
 Hello,
 
 Your OCF account has been disabled.
 
-Your account name is: ${sorriedUser}
+Your account name is: {0}
 
-The reason your account has been disabled:
-$(cat "$sorryFile")
+The reason your account has been disabled is:
+{1}
 
 Feel free to reply to this message.
 Please do not share your password with anyone or over email.
 
-$(python3 -c 'import ocflib.misc.mail as mail; print(mail.MAIL_SIGNATURE)')
-EOF
+{2}
+    '''.format(username, reason, ocflib.misc.mail.MAIL_SIGNATURE)
+    ocflib.misc.mail.send_mail(sorried_user_email, email_subject, email_body, cc=None)
+
+
+def main():
+    sorry_description = textwrap.dedent('''
+        Sorry a user.
+        Standard sorry reasons are:
+        {0}
+        Nevertheless, you can also pass in your own file with its file name as argument sorry_reason.
+    '''.format('\n'.join([f for f in os.listdir(REASON_PATH) if os.path.isfile(os.path.join(REASON_PATH, f))])))
+    parser = argparse.ArgumentParser(usage='%(prog)s sorried_user sorry_reason',
+                                     formatter_class=argparse.RawDescriptionHelpFormatter,
+                                     description=sorry_description)
+    parser.add_argument('sorried_user', metavar='sorried_user', help='Username of sorried user')
+    parser.add_argument('sorry_reason', metavar='sorried_reason', help='Reason for the user to be sorried')
+
+    if os.getuid() != 0:
+        raise PermissionError('You must be root to run this.')
+
+    args = parser.parse_args()
+    file_path = find_sorry_file(args.sorryReason)
```

Comment:
```
sorryReason
```

Shouldn't this be `sorry_reason`?

Reply:
Let me double check, i moved some code from `main()` to there